app/controllers/steps/client/has_nino_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Steps
+  module Client
+    class HasNinoController < Steps::ClientStepController
+      def edit
+        @form_object = HasNinoForm.build(
+          current_crime_application
+        )
+      end
+
+      def update
+        update_and_advance(HasNinoForm, as: :has_nino)
+      end
+
+      private
+
+      def decision_tree_class
```

Comment:
Although completely harmless, there is no need to declare the decision tree in this controller, as it will be using the one implemented in the superclass `Steps::ClientStepController` 👍

Reply:
Thanks don't know why I didn't spot that! :)